lib/find.js

```diff
@@ -8,59 +8,89 @@ const fs = require('fs'),
 
 /**
  * Find all registered serial ports by-id
- * @param  {Function} callback
- * @return {Promise}
+ *
+ * @return {Promise<Object[]|Error>}
  */
 function find () {
-
+  const serialPath = this.paths.linux.serial;
```

Comment:
I have a different use case. I use this for an embedded device. People that need windows support can use node-serialport.